src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -238,7 +238,7 @@ protected StorageEnvironmentOptions(VoronPathSetting tempPath, IoChangesNotifica
         public void SetCatastrophicFailure(ExceptionDispatchInfo exception)
         {
             _catastrophicFailure = exception;
-            _catastrophicFailureNotification.RaiseNotificationOnce(_environmentId, ToString(), exception.SourceException);
+            _catastrophicFailureNotification.RaiseNotificationOnce(_environmentId, ToString(), exception.SourceException, Environment.StackTrace);
```

Comment:
Why do we need to use `Environment.Stacktrace`? Wouldn't it be the same as `ExceptionDispatchInfo.SourceException` that we have here (`exception.SourceException`) ?

Reply:
That capture the case _until_ this point, with `Environment.Stacktrace` catching _From_ this point